heartbeat/monitors/active/dialchain/tls.go

```diff
@@ -21,9 +21,14 @@ import (
 	"net"
 	"time"
 
+	cryptoTLS "crypto/tls"
+
+	"fmt"
+
 	"github.com/elastic/beats/heartbeat/look"
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/outputs/transport"
+	"github.com/elastic/beats/packetbeat/protos/tls"
```

Comment:
I'm a bit concerned about this import. Beats should not have dependencies with each other. Share code like this be moved into libbeat.